test/factorization/par_ilu_kernels.cpp

```diff
@@ -78,6 +78,11 @@ class ParIlu : public CommonTestFixture {
             const auto row_start = row_ptrs[row];
             const auto row_end = row_ptrs[row + 1];
             const int num_row_elements = row_end - row_start;
+            if (num_row_elements == 0) {
```

Comment:
Btw. why do you have `int row` and `int num_row_elements` here while the input arguments are `index_type`?

Reply:
no particular reason beyond that nobody brought it up during review ;)